style.css

```diff
@@ -29,7 +29,7 @@ body {
   font-weight: bold;
```

Comment:
Great job @mesfin345 🤝

- [ ] As stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), kindly implement button interactions across all your buttons.
- [ ] Kindly make sure all your buttons are surrounded by `<a>` tags or `<button>` tags instead of `<p>` tags.

- [ ] Kindly remove the first button from your project as it makes your design different from the design provided in the Figma template.
- Your design
![your button first](https://user-images.githubusercontent.com/63917725/191692426-05479c18-f4d6-4977-8619-6011dd07703f.PNG)

- Figma design
![button figma](https://user-images.githubusercontent.com/63917725/191692480-54e4a3ab-c449-488b-921f-0ee52a0ad159.PNG)